src/App/App.js

```diff
@@ -1,5 +1,6 @@
 import React, {Component} from 'react'
 import Header from '../Header/Header'
+import Movies from '../Movies/Movies'
 import './App.css'
```

Comment:
I like the use of the directories to keep this organized. We are gonna thank ourselves later!

Reply:
That's what I call EXTREME ORGANIZATION